src/Microsoft.ML.StandardLearners/Standard/Online/AveragedPerceptron.cs

```diff
@@ -42,12 +40,21 @@ public sealed class AveragedPerceptronTrainer : AveragedLinearTrainer<BinaryPred
 
         public sealed class Options : AveragedLinearArguments
         {
+            /// <summary>
+            /// The custom <a href="tmpurl_loss">loss</a>. Default is hinge loss.
+            /// </summary>
             [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
             public ISupportClassificationLossFactory LossFunction = new HingeLoss.Arguments();
 
+            /// <summary>
+            /// The <a href="tmpurl_calib">calibrator</a> for producing probabilities. Default is exponential (aka Platt) calibration.
```

Comment:
>tmpurl_calib [](start = 29, length = 12)

one more

Reply:
these are by design: [https://github.com/dotnet/machinelearning/issues/2356](https://github.com/dotnet/machinelearning/issues/2356)

---
In reply to: [255267865](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255267865) [](ancestors = 255267865)